Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeUtil.java

```diff
@@ -169,58 +158,6 @@ private static BlackboardArtifact getCorrAttrSourceArtifact(BlackboardArtifact a
         return sourceArtifact;
     }
 
-    /**
-     * Makes a correlation attribute instance from a phone number attribute of an
-     * artifact.
-     *
-     * @param artifact An artifact with a phone number attribute.
-     *
-     * @return The correlation instance artifact or null, if the phone number is
-     *         not a valid correlation attribute.
-     *
-     * @throws TskCoreException     If there is an error querying the case
-     *                              database.
-     * @throws CentralRepoException If there is an error querying the central
-     *                              repository.
-     */
-    private static CorrelationAttributeInstance makeCorrAttrFromArtifactPhoneAttr(BlackboardArtifact artifact) throws TskCoreException, CentralRepoException {
-        CorrelationAttributeInstance corrAttr = null;
-
-        /*
-         * Extract the phone number from the artifact attribute.
-         */
-        String value = null;
-        if (null != artifact.getAttribute(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_PHONE_NUMBER))) {
-            value = artifact.getAttribute(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_PHONE_NUMBER)).getValueString();
-        } else if (null != artifact.getAttribute(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_PHONE_NUMBER_FROM))) {
-            value = artifact.getAttribute(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_PHONE_NUMBER_FROM)).getValueString();
-        } else if (null != artifact.getAttribute(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_PHONE_NUMBER_TO))) {
-            value = artifact.getAttribute(new BlackboardAttribute.Type(BlackboardAttribute.ATTRIBUTE_TYPE.TSK_PHONE_NUMBER_TO)).getValueString();
-        }
-
-        /*
-         * Normalize the phone number.
-         */
-        if (value != null) {
-            String newValue = value.replaceAll("\\D", "");
-            if (value.startsWith("+")) {
-                newValue = "+" + newValue;
-            }
-            value = newValue;
-
-            /*
-             * Validate the phone number. Three to five digit phone numbers may
-             * be valid, but they are too short to use as correlation
-             * attributes.
-             */
-            if (value.length() > 5) {
-                corrAttr = makeCorrAttr(artifact, CentralRepository.getInstance().getCorrelationTypeById(CorrelationAttributeInstance.PHONE_TYPE_ID), value);
-            }
-        }
```

Comment:
It seems like this normalization should be retained in a method to be called by makeCorrAttrFromAcctArtifact when it process a TSK_ACCOUNT artifact for a phone account.

Reply:
The CommunicationsManager is already doing the same normalization on phone numbers before creating accounts. Do we really want to repeat that?

W.r.t the length check, my thought was that since we are now only creating these attributes for actual phone numbers which have been retrieved from call logs or messages, and not from Keyword searches, the odds of getting short false positives are much less.  On the other hand, we may be dropping on the floor real short codes such as 411/611/911.

I can certainly put it all back in if you prefer that.

Reply:
I f we have it covered elsewhere, no worries.

Reply:
Just to chime in here, there is no length check in the CommManager. Also, keyword searches were calling makeCorrAttrFromArtifactAttr(), not this function. I'm not sure if any of this matters. It seems important to know if 911 was called, but maybe everyone in the CR has called 911 so the original comment stands. In any case, the code is still picking up the phone numbers that appear in CONTACT, MESSAGE, and CALLOG artifacts, so I'm going to approve.